arches/management/commands/packages.py

```diff
@@ -1114,6 +1124,128 @@ def import_reference_data(self, data_source, overwrite="ignore", stage="stage",
         skos = SKOSReader()
         rdf = skos.read_file(data_source)
         ret = skos.save_concepts_from_skos(rdf, overwrite, stage, prevent_indexing)
+    
+    def import_controlled_lists(self, source):
+        if os.path.exists(source):
+            wb = openpyxl.load_workbook(source)
+            with transaction.atomic():
+                for sheet in wb.sheetnames:
+                    if sheet == "ControlledList":
+                        self.import_sheet_to_model(wb[sheet], models.ControlledList)
+                    elif sheet == "ControlledListItem":
+                        self.import_sheet_to_model(wb[sheet], models.ControlledListItem)
+                    elif sheet == "ControlledListItemValue":
+                        self.import_sheet_to_model(wb[sheet], models.ControlledListItemValue)
+                # validate all data
+                for model in [models.ControlledList, models.ControlledListItem, models.ControlledListItemValue]:
+                    for instance in model.objects.all():
+                        instance.full_clean()
+                self.stdout.write('Data imported successfully from {0}'.format(source))
+        else:
+            self.stdout.write('The source file does not exist. Please rerun this command with a valid source file.')
+            sys.exit()
```

Comment:
```suggestion
```
Suggest letting django handle exiting, otherwise AFAICT you can't call this command from other code with `call_command()`.